samples/cellular/modem_shell/src/link/link_shell.c

```diff
@@ -1739,9 +1955,13 @@ static int link_shell_search(const struct shell *shell, size_t argc, char **argv
 	int opt;
 
 	while ((opt = getopt_long(argc, argv, short_options, long_options, NULL)) != -1) {
-		link_shell_getopt_common_option(opt, &common_option);
-
 		switch (opt) {
+		case 'r':
+		case LINK_SHELL_OPT_WRITE:
+		case LINK_SHELL_OPT_START:
```

Comment:
Should there  be `LINK_SHELL_OPT_RESET` as well?

Reply:
Yep. This is mapped from --clear.